src/YieldDisburser.sol

```diff
@@ -40,6 +47,10 @@ contract YieldDisburser is OwnableUpgradeable {
         for (uint256 i; i < _breadchainProjects.length; ++i) {
             breadchainProjects[i] = _breadchainProjects[i];
         }
+        minVotingAmount = 5; // must hold atleast 5 bread
+        minVotingHoldingDuration = 10 days; // must hold for atleast 10 days
+        minRequiredVotingPower = 1e18 * minVotingAmount * minVotingHoldingDuration / BLOCKTIME; // Holding 10 bread for 5 days , assuming a 5 second block time
+        maxVotes = 1e4;
```

Comment:
There is a getter\setter, this is the default value for the initialize but can be changed, because the variables are public there is an automatic getter. 

The best practice is to add them as variables for the initialize, I will open an issue to fix this down the line when finalizing the deployment script, so that it will read from a json file that will hold the values that we want to be sent to the initalizer 

Opened #42 , will resolve in a later pr